src/index.tsx

```diff
@@ -1,10 +1 @@
-import * as React from "react";
-import * as ReactDOM from "react-dom";
-
-import { UserInterface } from "./ui";
-
-ReactDOM.render(
-  <UserInterface />,
-
-  document.getElementById("react-container")
-);
+export { UserInterface } from "./ui";
```

Comment:
Does this not imply we should just rename ui to index?

Reply:
Possibly, but we may want to export other components or models in the future alongside just the whole UI